scripts/cli.rb

```diff
@@ -19,17 +19,19 @@ def parse
       appliance_desc = "Repo URL containing appliance scripts and configs(COPY/LINK/TEMPLATE)"
       build_desc     = "Repo URL containing the build config and kickstart"
       manageiq_desc  = "Repo URL containing the main manageiq code"
+      vsphere_desc   = "Build only the vsphere image.  Defaults to false and builds all images."
 
       @options = Trollop.options do
         banner "Usage: build.rb [options]"
 
-        opt :type,        type_desc,     :type => :string,  :default => DEFAULT_TYPE, :short => "t"
-        opt :reference,   git_ref_desc,  :type => :string,  :default => DEFAULT_REF,  :short => "r"
-        opt :local,       local_desc,    :type => :boolean, :default => false,        :short => "l"
-        opt :fileshare,   share_desc,    :type => :boolean, :default => true,         :short => "s"
-        opt :appliance_url, appliance_desc,  :type => :string,  :default => APPLIANCE_URL, :short => "A"
-        opt :build_url, build_desc, :type => :string,  :default => BUILD_URL, :short => "B"
-        opt :manageiq_url, manageiq_desc, :type => :string,  :default => MANAGEIQ_URL, :short => "M"
+        opt :type,          type_desc,      :type => :string,  :short => "t", :default => DEFAULT_TYPE
+        opt :reference,     git_ref_desc,   :type => :string,  :short => "r", :default => DEFAULT_REF
+        opt :local,         local_desc,     :type => :boolean, :short => "l", :default => false
+        opt :fileshare,     share_desc,     :type => :boolean, :short => "s", :default => true
+        opt :appliance_url, appliance_desc, :type => :string,  :short => "A", :default => APPLIANCE_URL
+        opt :build_url,     build_desc,     :type => :string,  :short => "B", :default => BUILD_URL
+        opt :manageiq_url,  manageiq_desc,  :type => :string,  :short => "M", :default => MANAGEIQ_URL
+        opt :vsphere_only,  vsphere_desc,   :type => :boolean, :short => "V", :default => false
```

Comment:
Oh I just saw the --targets comment from @abellotti .  I didn't realize it was already suggested :)